chain-signatures/node/src/indexer.rs

```diff
@@ -255,7 +256,7 @@ async fn handle_block(
                     epsilon,
```

Comment:
TODO: since the goal is to decouple Indexers from the Node, we can move the epsilon calculation to the node and avoid executing this logic in multiple Indexer implementations.